src/test/test_http_client_builder.rs

```diff
@@ -0,0 +1,12 @@
+use reqwest::{
+    header::{self, HeaderMap, HeaderValue},
+    ClientBuilder,
+};
+
+/// Create a http client builder that with some default headers
+pub fn test_http_client() -> ClientBuilder {
+    reqwest::ClientBuilder::new().default_headers(HeaderMap::from_iter([(
+        header::USER_AGENT,
+        HeaderValue::from_static("infra-smoke-test"),
```

Comment:
Let's change this to `rust-lang/infra-smoke-tests` so that it matches the repository name. That's been an (unofficial?) best practice, at least within the `rust-lang` organization. 🙂